src/distilabel/utils/serialization.py

```diff
@@ -194,7 +201,7 @@ def _model_dump(self, obj: Any, **kwargs: Any) -> Dict[str, Any]:
         """
         # Any parameter named api_key or token will be excluded from the dump
         # (those are supposed to be SecretStr anyway, and will remove them afterwards)
-        dump = obj.model_dump(exclude=("api_key", "token"), **kwargs)
+        dump = obj.model_dump(exclude="api_key", **kwargs)
```

Comment:
I've left "api_key" in for now given almost every test would need updated, focusing on proving the exception from "token" one would have been solved.